assets/docref/docref.js

```diff
@@ -21,7 +21,8 @@ $(function() {
       // Wait 250ms and scroll to anchor after loading
       setTimeout(function() {
         var hash = (window.location.hash || "").substr(1);
-        if(hash && hash !== "") {
+        // be a little careful about what gets interpolated into the jQuery here
+        if(hash && hash.match(/[a-zA-Z0-9_ -]+/)) {
```

Comment:
You're right -- `match` anchors implicitly at the beginning, but not at the end.